WMF Framework/WMFExploreFeedContentController.m

```diff
@@ -200,11 +201,7 @@ - (void)updateFeedSourcesWithDate:(nullable NSDate *)date userInitiated:(BOOL)wa
                                         [moc performBlock:^{
                                             NSError *saveError = nil;
                                             if ([moc hasChanges]) {
-                                                if (date) {
```

Comment:
@nambatee do you remember why the `if (date)` switch is here?

Reply:
I think it's because originally we were applying preferences to the updated objects when feed was updated but then we saw that it wasn't always working (in some cases, we wanted to apply preferences to all objects on feed update) so this date check achieved that without always fetching all objects